comms/dht/src/crypt.rs

```diff
@@ -88,6 +144,27 @@ pub fn encrypt(cipher_key: &CipherKey, plain_text: &[u8]) -> Vec<u8> {
     buf
 }
 
+/// Produces authenticated encryption of the signature using the ChaCha20-Poly1305 stream cipher,
+/// refer to https://docs.rs/chacha20poly1305/latest/chacha20poly1305/# for more details.
+/// Attention: as pointed in https://github.com/tari-project/tari/issues/4138, it is possible
+/// to use a fixed length Nonce, with homogeneous zero data, as this does not incur any security
```

Comment:
Not a fixed-length nonce, but a fixed nonce.

Reply:
Definitely ! Thank you !